controller/proxy-injector/webhook.go

```diff
@@ -40,6 +41,13 @@ func Inject(
 	if err != nil {
 		return nil, err
 	}
+
+	caPEM, err := ioutil.ReadFile(pkgK8s.MountPathTrustRootsPEM)
+	if err != nil {
+		return nil, err
```

Comment:
In `identity/main.go` an error is returned that is a little more descriptive: `log.Fatalf("could not read identity trust anchors PEM: %s", err.Error())`.

It could be worth doing some extra formatting here as well.